packages/next/client/image.tsx

```diff
@@ -156,11 +156,14 @@ export default function Image({
   // If priority and lazy are present, log an error and use priority only.
   if (priority && lazy) {
     if (process.env.NODE_ENV !== 'production') {
-      console.error(
-        `Image with src ${src} has both priority and lazy tags. Only one should be used.`
+      throw new Error(
```

Comment:
I noticed you throw instead of log error. Do we also want to throw on line 255?

Reply:
Yep!

Reply:
Done!